easy_rec/python/input/input.py

```diff
@@ -36,7 +40,8 @@ def __init__(self,
                input_path,
                task_index=0,
                task_num=1,
-               check_mode=False):
+               check_mode=False,
+               fg_json_path=None):
```

Comment:
add a parameter here requires to change all its subclasses, a better choice is to add an member function: set_fg_path(fg_json_path), and call it in _get_input_fn before create_input.

Reply:
Done.